### PR TITLE
Upgrade github.com/abcxyz/pkg to v0.7.0

### PR DESCRIPTION
**Pull request by lock14**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,263 +62,263 @@ jobs:
   go_lint:
     uses: 'abcxyz/pkg/.github/workflows/go-lint.yml@main' # ratchet:exclude
     with:
-      go_version: '1.20'
+      go_version: '1.21'
```
Comment (drevell) [inline .github/workflows/ci.yml:65]: Should these use `${{env.go_version}}` rather than hardcoding 1.21 (throughout this file)?

**Review by drevell — DISMISSED** [review]

File: .github/workflows/ci.yml
Comment (verbanicm) [inline .github/workflows/ci.yml:1]: Can you format this again? There should be 2 spaces per indent, not sure what happened here....

Comment (lock14) [reply .github/workflows/ci.yml:1]: done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (verbanicm) [reply .github/workflows/ci.yml:64]: I believe capitalization matters, the env is UPPER and you gave lower, maybe try that?

**Review by drevell — APPROVED** [review]

**Pull request merged**